PocketKit/Sources/PocketKit/Article/ReadableViewModel/RecommendableItemViewModel.swift

```diff
@@ -61,6 +61,14 @@ class RecommendableItemViewModel: ReadableViewModel {
         self.savedItemCancellable = item.publisher(for: \.savedItem).sink { [weak self] savedItem in
             self?.update(for: savedItem)
         }
+
+        if let url = item.shortURL {
+            self.shortUrl = url
+        } else {
+            Task {
+                self.shortUrl = try? await source.getItemShortUrl(item.givenURL)
```

Comment:
Is there a risk of no short url by the time the user saves?

Reply:
see slack discussion, short answer: probably not a lot of real world cases, except offline users sharing via airdrop.